In-house patch to add 'uar' and 'zfs' container and disk formats to
registered image types.  Patch has not yet been submitted upstream.

--- glance-12.0.0/glance/common/config.py.~1~	2016-04-07 00:37:11.000000000 -0700
+++ glance-12.0.0/glance/common/config.py	2016-06-27 18:46:02.569983220 -0700
@@ -42,14 +42,14 @@ paste_deploy_opts = [
 ]
 image_format_opts = [
     cfg.ListOpt('container_formats',
-                default=['ami', 'ari', 'aki', 'bare', 'ovf', 'ova', 'docker'],
+                default=['ami', 'ari', 'aki', 'bare', 'ovf', 'ova', 'docker', 'uar'],
                 help=_("Supported values for the 'container_format' "
                        "image attribute"),
                 deprecated_opts=[cfg.DeprecatedOpt('container_formats',
                                                    group='DEFAULT')]),
     cfg.ListOpt('disk_formats',
                 default=['ami', 'ari', 'aki', 'vhd', 'vmdk', 'raw', 'qcow2',
-                         'vdi', 'iso'],
+                         'vdi', 'iso', 'zfs'],
                 help=_("Supported values for the 'disk_format' "
                        "image attribute"),
                 deprecated_opts=[cfg.DeprecatedOpt('disk_formats',
